### PR TITLE
text correction of some method's documentation

### PR DESCRIPTION
**Pull request by leonarf**
It seem that some documentation text were copy/paste. I tried to fix them, but maybe I didn't correctly read the function, and it doesn't do what I've written.
Feel free to overwrite my changes.

### CODE REVIEW

File: wikirate4py/api.py [hunk 1]
```diff
@@ -1386,7 +1386,7 @@ def update_relationship_metric_answer(self, **kwargs):
     def add_source(self, **kwargs):
         """add_source(url, title, company, report_type, year)
 
-        Updates and Returns an existing relationship metric answer
+        Adds and Returns a company's source
```
Comment (vasgat) [inline wikirate4py/api.py:1389]: replace "a company's source" with "a source"

File: wikirate4py/api.py [hunk 2]
```diff
@@ -1437,7 +1437,7 @@ def add_source(self, **kwargs):
     def update_source(self, **kwargs):
         """update_source(name, title, company, report_type, year)
 
-        Updates and Returns an existing relationship metric answer
+        Updates and Returns an existing company's source
```
Comment (vasgat) [inline wikirate4py/api.py:1440]: replace "a company's source" with "a source"

**Review by vasgat — CHANGES_REQUESTED** [review]

**Comment by leonarf** [comment]
I made the requested changes.
By the way, I tried to connect to your slack, but it seems I need a wikirate mail adress :-(
Don't know if it's on purpose.

**Comment by vasgat** [comment]
Thanks Leonard, can you try again getting into slack? I just changed the settings.

**Pull request merged**